### ISSUE TITLE
Fixed circle stencil padding

### ISSUE THREAD

**Issue by KonradDeskiewicz** [Norserium/vue-advanced-cropper#46]
I've followed your documentation but cropper with fixed circle stencil has padding that doesn't allow  full image height or width to be used as a source of cropping. Didn't found (or don't understand) how to solve this issue.

Edit:
It looks that moving image around in fixed mode is not available until first shrink/zoom. After that `imageRestriction` seems to work fine.

**Comment by Norserium** [comment]
@KonradDeskiewicz, could you provide codesandbox that illustrates your problem?

**Comment by KonradDeskiewicz** [comment]
As you can see in the image below there is enough space to drag the image around with fixed circle stencil but you can't do that until you change a zoom a bit, When you back to initial zoom dragging is OK, but until zoom, it's not.

![image](https://user-images.githubusercontent.com/17060750/82452458-93550b80-9aaf-11ea-998f-3c1194810b49.png)

**Comment by KonradDeskiewicz** [comment]
And this is how the cropper is placed in component:

```javascript
<cropper
            ref="cropper"
            classname="cropper"
            background-classname="bg-primary-500"
            :src="img"
            :stencil-component="$options.components.CircleStencil"
            :stencil-props="{
              handlers: {},
              movable: false,
              scalable: false,
              aspectRatio: 1,
            }"
            image-restriction="stencil"
          />
```

**Comment by Norserium** [comment]
@KonradDeskiewicz, thank you for your report! This bug is fixed in `0.16.5`.

**Comment by KonradDeskiewicz** [comment]
Works fine :) Thanks for quick fix :)
